src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -943,8 +943,8 @@ public void StartInfo_WebPage()
                 UseShellExecute = true,
                 FileName = @"http://www.microsoft.com"
             };
-
-            Process.Start(info); // Returns null after navigating browser
+            // Returns null after navigating browser
+            using (var p = Process.Start(info)) { }
```

Comment:
I'm confused.  The comment says that this returns null.  Why do we then need a using?  Shouldn't we instead Assert.Null?